src/image_occlusion_enhanced/ngen.py

```diff
@@ -88,6 +90,8 @@ def __init__(self, ed, svg, image_path, opref, tags, fields, did):
         self.did = did
         self.qfill = '#' + mw.col.conf['imgocc']['qfill']
         loadConfig(self)
+        # set working directory to collection.media
+        os.chdir(media_paths_from_col_path(mw.col.path)[0])
```

Comment:
@zjosua just in case you run into the same issue type with other add-ons (as many probably still assume that the PWD is set): I would recommend against changing the directory globally as that could potentially break other add-ons that are now assuming the opposite (though I'd hope that most if not all have just switched away from making any assumptions about the working directory at all)